0011-revise-e2e-test-framework.adoc

```diff
@@ -0,0 +1,74 @@
+= Revise e2e test framework
+Tobin C. Harding <tobin.harding@coblox.tech>
+:toc:
+:revdate: 2019-07-29
+
+* Authors(s): {authors}
+* Date: {revdate}
+
+Issue: https://github.com/comit-network/comit-rs/issues/897[#985]
+
+== Context
+
+No one is happy with the current state of the end to end tests.
+This spike will attempt to ascertain what exactly is wrong with the current end to end tests.
+
+== Issues
+
+These are the issues distilled from talking with all team members (excluding Lucas).
+
+=== General
+
+* Tests are not independent.
+* As a developer, I want to be able to run a test against a comit_node/btsieve/docker container that I started myself (e.g. I attached a debugger to cnd), without having to change JS code.
+
+=== Framework
+
+* Not overly happy with mocha but not unhappy either, no real specifics.
```

Comment:
Mocha does not allow parallel run of tests. Jest give you that for free.
We must move to parallel run as the number of tests will explode once we add a 4th asset (which is part of our quarter OKRs).